iocore/net/SSLUtils.cc

```diff
@@ -141,8 +142,9 @@ static int ssl_vc_index = -1;
 static ink_mutex *mutex_buf      = nullptr;
 static bool open_ssl_initialized = false;
 
-RecRawStatBlock *ssl_rsb                   = nullptr;
-static InkHashTable *ssl_cipher_name_table = nullptr;
+RecRawStatBlock *ssl_rsb = nullptr;
+typedef std::unordered_map<const char *, intptr_t> cipherTable;
```

Comment:
This hashes on the pointer value, not on the string value. I'm not sure that OpenSSL guarantees to give the same pointer values when you fish them out of the SSL context.